parameter/ConfigurableElement.h

```diff
@@ -29,6 +29,8 @@
  */
 #pragma once
 
+#include <parameter_export.h>
```

Comment:
Coding style says that header should be included from closest to farthest. Eg: local, ... std header. I would recommend including with `""`, not `<>`.